forge-gui/res/cardsfolder/upcoming/claim_jumper.txt

```diff
@@ -0,0 +1,11 @@
+Name:Claim Jumper
+ManaCost:2 W
+Types:Creature Rabbit Mercenary
+PT:3/3
+K:Vigilance
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | CheckSVar$ X | SVarCompare$ GTY | OptionalDecider$ You | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ When CARDNAME enters the battlefield, if an opponent controls more lands than you, you may search your library for a Plains card, put it onto the battlefield tapped, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Plains | ChangeNum$ 1 | Tapped$ True | SubAbility$ DBChangeZone
```

Comment:
needs extra params so you only shuffle once in total in all cases?

Reply:
Best template I've come across so far are [https://scryfall.com/card/c13/108/from-the-ashes](https://scryfall.com/card/c13/108/from-the-ashes) and [Wave of Vitriol](https://scryfall.com/card/cma/165/wave-of-vitriol). Now my question is whether the `RememberChanged$ True` that links to the `ConditionCheckSVar$` on the `DB$ Shuffle` subability accounts for "failing to find".

Reply:
I checked it along with the other requirements: as it is it doesn't consider "failing to find". Otherwise, the new script passes the "shuffling once" test with flying colors.

Reply:
yea probably not 100% perfect, I'll think about it later
I think the extra Branch effect is unneeded though

Reply:
Fair.
Sure enough the `Branch` effect it isn't needed. Edited it accordingly.